src/arrays/referencing_array_and_its_elements/medium/q9/EveryEven.java

```diff
@@ -0,0 +1,37 @@
+package arrays.referencing_array_and_its_elements.medium.q9;
+
+/*
+Make a Java program that initializes an integer array with 10 elements, whose values are randomly generated between 1
```

Comment:
Change to:
Write a Java program that initializes an array of 10 integers randomly generated between 1 and 1000 (inclusive on both ends) and then prints the array. Then initialize a second array to be half the size of the first. Then copy all the values at even indexes from the first array and store them in the second array. Then print out the second array.